### PR TITLE
Use explicit (preferred) form for numeric literals

### PR DESCRIPTION
**Pull request by ericvaandering**
See https://www.python.org/dev/peps/pep-3127/#abstract

This turns up a couple of typos related to dates and to ROOT but will not change the results.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: CalibTracker/SiStripDCS/test/ParseManualO2Olog.py [hunk 1]
```diff
@@ -1308,7 +1308,7 @@ def CompareReducedDetIDs(FirstDict,SecondDict):
         #    print row
         #if row['change_date']==datetime.datetime(2010, 8, 28, 22, 51, 8, 994000):
         #    print row
-        if row['change_date']>datetime.datetime(2010, 8, 28, 22, 44) and row['change_date']<datetime.datetime(2010, 8, 28, 23, 05,37):
+        if row['change_date']>datetime.datetime(2010, 8, 28, 22, 44) and row['change_date']<datetime.datetime(2010, 8, 28, 23, 0o5,37):
```
Comment (diguida) [inline CalibTracker/SiStripDCS/test/ParseManualO2Olog.py]: this should be `5`, `datetime` accepts integers.

Comment (diguida) [inline CalibTracker/SiStripDCS/test/ParseManualO2Olog.py]: well, the representation does not matter. The actual value is the same...

````
>>> oct(5)
'05'
````

It was a misuse of the octal representation...

**Comment by ericvaandering** [comment]
I've updated the few places where dates were being given as 03 which was converted to 0o3. Doesn't change anything but does look less confusing.

File: Alignment/MuonAlignmentAlgorithms/scripts/findQualityFiles.py [hunk 2]
```diff
@@ -65,13 +65,13 @@
...
                    type="int",
-                   default=0L,
+                   default=0,
                    dest="startRun")
 
 parser.add_option("-e", "--endRun",
                    help="Last run number in range.",
                    type="int",
-                   default=999999999L,
+                   default=999999999,
```
Comment (Dr15Jones) [inline Alignment/MuonAlignmentAlgorithms/scripts/findQualityFiles.py]: Doesn't this undo a change you made?

**Comment by Dr15Jones** [comment]
Please test

File: Alignment/MuonAlignmentAlgorithms/scripts/findQualityFiles.py (hunk 2 above)
Comment (ericvaandering) [inline Alignment/MuonAlignmentAlgorithms/scripts/findQualityFiles.py]: No. No such thing as a long in Python anymore.

**Pull request merged**